cloudca/resource_cloudca_port_forwarding_rule_test.go

```diff
@@ -13,9 +13,6 @@ import (
 func TestAccPortForwardingRuleCreate(t *testing.T) {
 	t.Parallel()
 
-	environmentID := "c67a090f-b66f-42e1-b444-10cdff9d8be2"
-	vpcID := "2c01d952-d010-4811-b66d-4c7f5f805193"
-	networkID := "405e35c3-3e69-4e02-a162-a4112d94acd9"
```

Comment:
These are intended behavior of having acceptance tests, they have to physically run on an actual organization and environment. You should not remove these hardcoded values.

Reply:
these were changed to values that worked with the organization and environment we have (and moved to be global since the linter was complaining)